frontend/src/lib/components/InstanceSettings.svelte

```diff
@@ -213,8 +214,9 @@
 								<OAuthSetting name="gitlab" bind:value={oauths['gitlab']} />
 								<OAuthSetting name="jumpcloud" bind:value={oauths['jumpcloud']} />
 								<KeycloakSetting bind:value={oauths['keycloak']} />
+								<AuthentikSetting bind:value={oauths['authentik']} />
 								{#each Object.keys(oauths) as k}
-									{#if !['google', 'microsoft', 'github', 'gitlab', 'jumpcloud', 'okta', 'keycloak', 'slack'].includes(k) && 'login_config' in oauths[k]}
+									{#if !['authentik','google', 'microsoft', 'github', 'gitlab', 'jumpcloud', 'okta', 'keycloak', 'slack'].includes(k) && 'login_config' in oauths[k]}
```

Comment:
missing a similar check below

Reply:
line 284

Reply:
I think you meant line 284 and as such, it's done. :)